src/components/movies-component/movies-component.tsx

```diff
@@ -0,0 +1,12 @@
+import { MoviesContainer } from "components/movies-container/movies-container"
+
+import styles from './movies-component.module.scss'
+
+export const MoviesComponent = () => {
+    return (
+        <>
+        <div className={styles.devider}/>
```

Comment:
1. Let's use tags tabulation to make it easier to read. It would be also great if you update linter to apply these changes.
2. Divider*